lib/src/model/movie.dart

```diff
@@ -0,0 +1,29 @@
+import 'result.dart';
+
+class Movie {
+  int _page = 0;
+  int _totalResults = 0;
+  int _totalPages = 0;
+  List<Result> _results = [];
+
+  Movie.fromJson(Map<String, dynamic> parsedJson) {
+    print(parsedJson['results'].length);
+    _page = parsedJson['page'];
+    _totalResults = parsedJson['total_results'];
+    _totalPages = parsedJson['total_pages'];
+    List<Result> temp = [];
+    for (int i = 0; i < parsedJson['results'].length; i++) {
+      Result result = Result(parsedJson['results'][i]);
+      temp.add(result);
+    }
+    _results = temp;
```

Comment:
you should avoid shortening words, in "temp" you can use a more representative word like "temporalList" or something like that

Reply:
Fixed